src/jupyter_contrib_nbextensions/nbextensions/code_prettify/code_prettify.js

```diff
@@ -41,7 +41,7 @@ define(function(require, exports, module) {
             "postfix": ", output=FALSE)[['text.tidy']], collapse='\n')))"
         },
         "javascript": {
-            "library": "jsbeautify = require(" + "'js-beautify')",
+            "library": "jsbeautify = requirejs(" + "'js-beautify')",
```

Comment:
not sure about this, it's a javascript _kernel_ which is running this require call, so it likely _doesn't_ want requirejs?

Reply:
Good catch! Correct.